src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testPieces;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StartButtonTest {
+
+    @Test
+    public void isPressed() {
+        StartButton button = new StartButton();
+        boolean expected = true;
+        boolean actual = button.isPressed();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getButtonPressed() {
+        StartButton button = new StartButton();
+        boolean expected = false;
+        boolean actual = button.getButtonPressed();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setButtonState() {
```

Comment:
the PR has to follow our convention. you need to change the name of method

Reply:
Okay thanks for the observation

Reply:
Already done